mars/deploy/oscar/session.py

```diff
@@ -1174,7 +1181,13 @@ async def fetch_infos(self, *tileables, fields, **kwargs) -> list:
         if kwargs:  # pragma: no cover
             unexpected_keys = ", ".join(list(kwargs.keys()))
             raise TypeError(f"`fetch` got unexpected arguments: {unexpected_keys}")
-
+        need_fetch_fields = {"level", "memory_size", "store_size"}
+        need_fetch = bool(need_fetch_fields & fields)
```

Comment:
What does the `need_fetch` mean?

Reply:
Some chunk info such as `"level", "memory_size", "store_size"` needs to access `storage API`. For those fields, we mark them as `need_fetch`. Maybe `need_query_storage` better?

Reply:
Better to use a clear name or add some comments.